src/lib/dnssd/minimal_mdns/AddressPolicy_BuiltinImpl.h

```diff
@@ -16,10 +16,12 @@
  */
 #pragma once
 
+#include <lib/dnssd/minimal_mdns/AddressPolicy.h>
+
 namespace mdns {
 namespace Minimal {
 
-void SetDefaultAddressPolicy();
+AddressPolicy * GetDefaultAddressPolicy();
```

Comment:
This should probably be removed from here. Likely we may not need this header at all.